metadata/src/test/java/org/apache/kafka/controller/ReplicationControlManagerTest.java

```diff
@@ -491,7 +491,7 @@ void alterTopicConfig(
         }
 
         void fenceBrokers(Integer... brokerIds) {
-            fenceBrokers(Utils.mkSet(brokerIds));
+            fenceBrokers(Set.of(brokerIds));
```

Comment:
The failure is caused by the undefined order. I file https://issues.apache.org/jira/browse/KAFKA-17772 to fix it for now.